vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -157,6 +205,14 @@
       <Project>{2d5d43d9-cffc-4c40-b4cd-02efb4e2742b}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)' == 'ReleaseBundle' or '$(Configuration)' == 'DebugBundle'">
+    <CustomBuildStep>
+      <Command>npm run bundle</Command>
+      <Outputs>$(ProjectDir)Bundle\index.windows.bundle;$(ProjectDir)Bundle\assets</Outputs>
```

Comment:
>$(ProjectDir)Bundle\assets [](start = 56, length = 26)

why do you need this?

Reply:
in js you can require assets (jpg png etc) like require(..\image.jpg) when the bundler runs it will locate those assets and copy them to the assets dir specified in the bundle command.
The intention is to lay them out so that they will be in the correct relative directory at runtime

Reply:
I've created a bundle command that packages all assets into the assets directory.  This seems sufficient for basic scenarios, but if the developer wanted to run the bundle command themselves and put assets somewhere else, we'd need to do more to package them into the app.  This seems like a good first start (unless I'm missing a basic requirement).

---
In reply to: [333598505](https://github.com/microsoft/react-native-windows/pull/3367#discussion_r333598505) [](ancestors = 333598505)

Reply:
I found that without this, the bundle and/or assets were sometimes not getting copied into the output directory.

---
In reply to: [333192287](https://github.com/microsoft/react-native-windows/pull/3367#discussion_r333192287) [](ancestors = 333192287)